File: python/2.py
def fibonacciEvenSum(max: int) -> int:
    sum = 0
    a = 1
    b = 2

    while a < max:
        if a % 2 == 0:
            sum += a

        a, b = b, a + b

    return sum

print(fibonacciEvenSum(4e6))

File: typescript/2.ts
let sum = 0;
let a = 1;
let b = 2;

while (a < 4e6) {
    if (a % 2 == 0) {
        sum += a;
    }

    [a, b] = [b, a + b];
}

console.log(sum);
// 4613732

export {}
